mathics/doc/documentation/1-Manual.mdoc

```diff
@@ -168,8 +168,8 @@ Of course, \Mathics has complex numbers:
 ('!' denotes the factorial function.)
 The precision of numerical evaluation can be set:
 
-  >> N[Pi, 100]
-  = 3.141592653589793238462643383279502884197169399375105820974944592307816406286208998628034825342117068
```

Comment:
Maybe a test that is expected to fail would be nice here, to detect when the regression is fixed

Reply:
@TiagoCavalcante Just so you know...

The problem right now that @mmatera isolated is that the function `mp_constant`  does not reset the precision after doing something like this. And then subsequent functions that use mpmath get very very slow.

We see this in testing where a 2 minute test jumps to 10 minutes, after that mpmath precision has gotten set to 100 or so. 

@mmatera offers: 

```
mathics -e 'p=N[Pi,100];' -e 'Timing[Plot[BesselK[1,x],{x,0,2}]]'
```

as an example to try to show the problem. 

He is currently working on a fix to this.  

I think a test to show that the precision is reset is fine. However this would _not_ be a doctest but rather a pytest.  

And I imagine that will appear along with the code to fix. The code above is just a temporary workaround.